input/otlp/logs_test.go

```diff
@@ -118,14 +203,14 @@ func TestConsumerConsumeLogs(t *testing.T) {
 			}
 
 			expected := proto.Clone(&commonEvent).(*modelpb.APMEvent)
-			expected.Message = expectedMessage
+			expected.Message = tt.expectedEvent.Message
+			if tt.expectedEvent.Session != nil {
+				expected.Session = tt.expectedEvent.Session
```

Comment:
This is a bit manual/verbose. But since we don't currently need more fields to be customized from the default base event, it's still simpled than using reflect to merge both structs.